ajax/libs/ideal-image-slider/package.json

```diff
@@ -14,15 +14,14 @@
     "type": "git",
     "url": "https://github.com/gilbitron/Ideal-Image-Slider.git"
   },
-  "npmName": "ideal-image-slider",
-  "npmFileMap": [
-    {
-      "basePath": "",
-      "files": [
-        "ideal-image-slider*",
-        "extensions/**/*.!(md)",
-        "themes/**/*"
-      ]
-    }
-  ]
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/gilbitron/Ideal-Image-Slider.git",
+    "basePath": "",
+    "files": [
+      "ideal-image-slider*",
+      "extensions/**/*.!(md)",
+      "themes/**/*"
+    ]
+  }
```

Comment:
There is no need to transfer to git auto-update in the config, you are just adding the missing old versions.